From 4c6c1ffcda99f294245fdf220cc5f01a5cf5d7f6 Mon Sep 17 00:00:00 2001
From: Danny Nold <dannynold@freescale.com>
Date: Mon, 16 May 2011 10:59:18 -0500
Subject: [PATCH 967/998] ENGR00143486 - MSL: MAX17135 init_data incorrectly declared __initdata

- Removed __initdata from regulator_init_data structure declaration.  This
protects the MAX17135 regulator from having its structure overwritten, which
was causing a bug when writing large chunks of memory.

Signed-off-by: Danny Nold <dannynold@freescale.com>
---
 arch/arm/mach-mx5/mx50_rdp.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/mach-mx5/mx50_rdp.c b/arch/arm/mach-mx5/mx50_rdp.c
index 05f7fcf..b4662a5 100644
--- a/arch/arm/mach-mx5/mx50_rdp.c
+++ b/arch/arm/mach-mx5/mx50_rdp.c
@@ -666,7 +666,7 @@ static struct mxc_spi_master mxcspi3_data = {
 #define V_to_uV(V) (mV_to_uV(V * 1000))
 #define uV_to_V(uV) (uV_to_mV(uV) / 1000)
 
-static struct regulator_init_data max17135_init_data[] __initdata = {
+static struct regulator_init_data max17135_init_data[] = {
 	{
 		.constraints = {
 			.name = "DISPLAY",
-- 
1.7.4.1

